testing/examples/WORKSPACE

```diff
@@ -16,9 +16,11 @@ workspace(name = "dockerfile_examples")
 
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive", "http_file")
 
-local_repository(
+http_archive(
```

Comment:
Why the change from local_repository? You should leave it as local repo otherwise you would be testing a committed version of rules_docker not the changes you have in the PR

Reply:
Done.
Changed that when had trouble running targets (on GCB) in the "internal" bazel project within the rules_docker project. I figured it out now.